README.md

```diff
@@ -121,16 +113,10 @@ for more information about how to generate these.
 
 ```puppet
 node 'broker1.example.com' {
-  class { '::mcollective':
-    middleware         => true,
-    middleware_hosts   => [ 'broker1.example.com' ],
-    middleware_ssl     => true,
```

Comment:
In this example, it's just a broker, so just a place where activemq goes. That's what the previous code did, it wouldn't install mcollective server or client components, just manage the middleware onto this host.